converters/toZigbee.js

```diff
@@ -1717,6 +1717,36 @@ const converters = {
             }
         },
     },
+    xiaomi_ZNCZ11LM_power_outage_memory: {
```

Comment:
Please merge this converter with https://github.com/Koenkk/zigbee-herdsman-converters/blob/0a757bf9787a2cbcaccef27a178beb260966b6a9/converters/toZigbee.js#L1723